hybrid/clusterlink/ccloud-as-source-cluster/README.md

```diff
@@ -58,10 +58,7 @@ kubectl -n source create secret generic source-tls-group1 \
 
 #### create password encoder secret
 ```
-
-    
-kubectl -n destination create secret generic password-encoder-secret \
-    --from-file=password_encoder_secret=$TUTORIAL_HOME/password-encoder-secret.txt
+ kubectl create secret generic password-encoder-secret --from-file=password-encoder.txt=password-encoder-secret.txt
```

Comment:
add -n destination. 

You need to update source-tls-group1 to have "-n desination". You don't have source namespace in this playbook.

Reply:
@karthikeyanas done

Reply:
@rahulbats I still see this 

```
### create secret for the certificates to connect to Confluent Cloud
```
kubectl -n source create secret generic source-tls-group1 \
    --from-file=fullchain.pem=$TUTORIAL_HOME/fullchain.pem \
    --from-file=cacerts.pem=$TUTORIAL_HOME/cacerts.pem 
    
```
```

Reply:
@karthikeyanas yeah fixed it now